apps/condo/domains/property/schema/Property.js

```
@@ -1,73 +1,84 @@
             isRequired: true,
         },
+
         address: {
+            // TODO(codegen): write doc for Property.address field!
+            schemaDoc: 'TODO DOC!',
             type: Text,
             isRequired: true,
-            // TODO(pahaz): need to checkIt!
         },
+
         addressMeta: {
+            // TODO(codegen): write doc for Property.addressMeta field!
+            schemaDoc: 'Property address components',
             type: Json,
             isRequired: true,
-            schemaDoc: 'Property address components',
             kmigratorOptions: { null: false },
             hooks: {
                 validateInput: ({ resolvedData, fieldPath, addFieldValidationError }) => {
                     if (!resolvedData.hasOwnProperty(fieldPath)) return addFieldValidationError(`${REQUIRED_NO_VALUE_ERROR}${fieldPath}] Value is required`)
                     const value = resolvedData[fieldPath]
-                    if (typeof value !== 'object' || value === null) {return addFieldValidationError(`${JSON_EXPECT_OBJECT_ERROR}${fieldPath}] ${fieldPath} field type error. We expect JSON Object`)}
+                    if (typeof value !== 'object' || value === null) { return addFieldValidationError(`${JSON_EXPECT_OBJECT_ERROR}${fieldPath}] ${fieldPath} field type error. We expect JSON Object`) }
                     const { dv } = value
                     if (dv === 1) {
                         // TODO(pahaz): need to checkIt!
                     } else {
-                        return addFieldValidationError(`${JSON_UNKNOWN_VERSION_ERROR}${fieldPath}] Unknown \`dv\` attr inside JSON Object`)
+                        // TODO(zuch): Need to understand what we are doing here
+                        console.error(`${JSON_UNKNOWN_VERSION_ERROR}${fieldPath}] Unknown \`dv\` attr inside JSON Object`)
+                        // return addFieldValidationError(`${JSON_UNKNOWN_VERSION_ERROR}${fieldPath}] Unknown \`dv\` attr inside JSON Object`)
```

We should throw an error in case of unknown `dv` (datastucture version)